app/src/main/java/org/simple/clinic/scanid/ScanSimpleIdUpdate.kt

```diff
@@ -0,0 +1,10 @@
+package org.simple.clinic.scanid
+
+import com.spotify.mobius.Next
+import com.spotify.mobius.Next.noChange
+import com.spotify.mobius.Update
+
+class ScanSimpleIdUpdate : Update<ScanSimpleIdModel, ScanSimpleIdEvent, ScanSimpleIdEffect> {
+  override fun update(model: ScanSimpleIdModel, event: ScanSimpleIdEvent): Next<ScanSimpleIdModel,
+      ScanSimpleIdEffect> = noChange()
```

Comment:
Why is this on the next line? 🤔

Reply:
Oh, I am using `mobius-creator` plugin and it seems to be auto-formatting code.